trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super.correct_name[0..9]
```

Comment:
- [ Required ] I think it would be nice if you refactor this line of code for this method to return the correct result. The `super` keyword refers to the superclass's method, in this case, `correct_name`, therefore there is no need to call `super.correct_name` you just need to call `super` as shown below.

## Example code
```ruby
require_relative 'decorator'

class TrimmerDecorator < Decorator
  def correct_name
    super[0..9]
  end
end
```

## Error when running the App
![Screenshot from 2023-10-12 00-20-47](https://github.com/ahmidou09/OOP-school-library/assets/16392046/82ddb73b-b850-4d3e-a0db-b27b5567f9aa)